src/Client.php

```diff
@@ -130,6 +131,18 @@ public function send(RequestInterface $request, array $options = [])
         return $this->sendAsync($request, $options)->wait();
     }
 
+    /**
+     * The HttpClient PSR (PSR-18) specify this method.
+     */
+    public function sendRequest(RequestInterface $request): ResponseInterface
```

Comment:
Thinking out loud here.. without having the answer on my head.. 

Should this be final?